### DISCUSSION TITLE
Slow sync - peer issue?

### DISCUSSION THREAD

**Discussion by leefarg** [Chia-Network/chia-blockchain#5333] [Support]
Many people have been reporting slow sync and loss of sync with unable to resync issues over the last several days. Some have postulated that perhaps it is a "bad peers" issue.

I'm currently very slowly resyncing after losing sync a few days ago. Tried all the usual - stopping node, completely shutting down, restarting computer, deleting mainnet db and restarting, etc. Running 1.1.5 Windows GUI (issue started on 1.1.3).

I notice on my list of connections, that the top one after mine has 0.1/465.7 MiB Up/down. A few others have 0.0/0.9, 0.0/0.5, etc. And lots are at 0/0.

Is this normal operation? Should I try deleting some of the 0/0 nodes and see what happens?

Thanks.

**Comment by Tibi38** [comment]
me the same, and trying in gui to use "connect to other peers"-Button is without any results.

**Reply by iMAGiCJAKE** [reply]
50~300 peak height behind forever.......... three days already...

**Reply by iMAGiCJAKE** [reply]
I think the issue is peers are not opened port 8444, can gui kills connection/peer who is not opened or no response in two seconds, or it is just wasting time.

**Reply by Tibi38** [reply]
the ports are open, because it was for more then 2 weeks in sync, problem starts behind 1.1.12 (or later)

**Comment by BrandtH22** [comment]
If you are still experiencing the issues it is recommended to upgrade to the latest version of chia. Please download it from here: https://www.chia.net/downloads/

Since this issue has been open for a number of years without additional comments, we will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions, comments, or if the issue persists after an update.

```
Please note that upgrading major versions of chia generally requires the config.yaml to be recreated using these steps:
1. Stop chia
2. Rename the config or move it to a different directory
3. In CLI run chia init
4. Copy any custom settings from the old config to the new (plot drive directories, pool info, trusted node info, etc) - be careful here as yaml is very picky about formatting including those leading spaces
5. Start Chia
```

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
